data/entities.json5

```diff
@@ -7523,6 +7523,11 @@
     "domains": ["*.movementventures.com", "*.qzzr.com"],
     "examples": ["www.qzzr.com"]
   },
+  {
+    "name": "RapidAPI",
+    "categories": ["utility"],
+    "domains": ["*.rapidapi.com"]
```

Comment:
any chance you've got an example? like `cdn.rapidapi.com` or just generally how these urls look? :)

Reply:
@patrickhulce they advertise "a directory of some 8,000 APIs". I only use one, called Telize, for geolocation and the URL looks like this, but I couldn't speak to all the others: `telize-v1.p.rapidapi.com`

Reply:
I can add that to the PR if you think it's worthwhile?

Reply:
yep that'd be great :)

Reply:
done https://github.com/patrickhulce/third-party-web/pull/101/commits/794adbd0843c749393c1b461987f109c73f49f8b

Reply:
thanks!